doc/en/customize.rst

```diff
@@ -39,8 +39,9 @@ Here's a summary what ``pytest`` uses ``rootdir`` for:
 influence how modules are imported. See :ref:`pythonpath` for more details.
 
 The ``--rootdir=path`` command-line option can be used to force a specific directory.
-The directory passed may contain environment variables when it is used in conjunction
-with ``addopts`` in a ``pytest.ini`` file.
+Note that contrary all other command-line options, ``--rootdir`` cannot be used with
+:confval:`addopts` inside ``pytest.ini`` because the ``rootdir`` is used to *find* ``pytest.ini``
+already.
```

Comment:
As for "all other": IIRC `--assertmode` also does not work there, but can also be fixed (IIRC).